optuna/samplers/_partial_fixed.py

```diff
@@ -1,8 +1,6 @@
 from __future__ import annotations
 
 from typing import Any
-from typing import Dict
-from typing import Optional
 from typing import Sequence
```

Comment:
`typing.Sequence` is deprecated. Could you replace it with `collections.abc.Sequence`?

```suggestion
from collections.abc import Sequence
```